apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/ErrorReporting/ErrorReportingExceptionLogger.cs

```diff
@@ -165,6 +143,7 @@ private ReportedErrorEvent CreateReportRequest(ExceptionLoggerContext context)
                 Message = context.Exception.ToString() ?? "",
                 Context = errorContext,
                 ServiceContext = _serviceContext,
+                EventTime = Timestamp.FromDateTime(DateTime.UtcNow),
```

Comment:
For testability you may want to use an IClock at some point, but that can come later.